test/run.js

```diff
@@ -0,0 +1,20 @@
+const spawn = require('child_process').spawn
+
+const {start, stop} = require('./server')
+
+start(() => {
+  const tape = spawn('tape', ['test/init.js', 'src/**/*.spec.js'])
+
+  tape.stdout.on('data', (data) => {
+    process.stdout.write(data)
```

Comment:
If we want to keep `spawn`, then `tape.stdout.pipe(process.stdout)` & `tape.stderr.pipe(process.stderr)`. Also, handle the `error` events.